### PR TITLE
feat: add create entity data table checks

### PR DESCRIPTION
**Pull request by BLasan**
## Description
Added Create Entity Data Table Checks component.

## Related issues and discussion
#751 

## Screenshots, if any
Screenshot from 2020-05-22 14-54-08

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by BLasan** [comment]
@karantakalkar Please review

File: src/app/organization/entity-data-table-checks/create-enity-data-table-checks/create-enity-data-table-checks.component.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      'datatableName': ['', Validators.required]
+    });
+  }
+
+  /**
+   * Get Entity on selection.
+   * @param entity Selected Entity.
+   */
+  getEntityType(entity: string) {
+    if (entity === 'm_client') {
```
Comment (karantakalkar) [inline src/app/organization/entity-data-table-checks/create-enity-data-table-checks/create-enity-data-table-checks.component.ts:74]: use switch case

File: src/app/organization/organization-routing.module.ts [hunk 2]
```diff
@@ -302,11 +304,24 @@ const routes: Routes = [
...
+              path: '',
+              component: EntityDataTableChecksComponent,
+              resolve: {
+                entityDataTableChecks: EntityDataTableChecksResolver
+              },
+            },
+            {
+              path: 'create',
+              component: CreateEnityDataTableChecksComponent,
+              data: { title: extract('Create Entity Data Table Checks'), breadcrumb: 'Create Entity Data Table Checks' },
```
Comment (karantakalkar) [inline src/app/organization/organization-routing.module.ts:319]: Bread Crumb too long
```suggestion
              data: { title: extract('Create Entity Data Table Checks'), breadcrumb: 'Create' },
```

**Comment by BLasan** [comment]
@karantakalkar  Done. Please review

**Comment by karantakalkar** [comment]
@BLasan resolve MCs for review.

**Comment by BLasan** [comment]
> @BLasan resolve MCs for review.

I'll resolve

**Comment by BLasan** [comment]
@karantakalkar Done

File: src/app/organization/entity-data-table-checks/create-enity-data-table-checks/create-enity-data-table-checks.component.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+    <form [formGroup]="createEntityForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <mat-form-field>
+            <mat-label>Entity</mat-label>
+            <mat-select required formControlName="entity" (selectionChange)="getEntityType($event.value)">
```
Comment (karantakalkar) [inline src/app/organization/entity-data-table-checks/create-enity-data-table-checks/create-enity-data-table-checks.component.html:13]: subscribe to value changes see create center form

File: src/app/organization/entity-data-table-checks/entity-data-table-checks.component.html [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
 
-  <button mat-raised-button color="primary">
+  <button mat-raised-button color="primary" [routerLink]="['create']">
     <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;
     Create Entity Data Table Check
```
Comment (karantakalkar) [inline src/app/organization/entity-data-table-checks/entity-data-table-checks.component.html:5]:
```suggestion
    Create
```

**Comment by BLasan** [comment]
@karantakalkar Please review

**Comment by karantakalkar** [comment]
resolve mcs please

**Comment by karantakalkar** [comment]
@BLasan Kindly remove MCs

**Comment by BLasan** [comment]
> @BLasan Kindly remove MCs

Done

**Comment by BLasan** [comment]
@karantakalkar Please review

**Comment by karantakalkar** [comment]
Yes will merge in sometime.

**Pull request merged**